src/client/README.md

```diff
@@ -0,0 +1,6 @@
+# client/
```

Comment:
Everything below is all the code from the separate api client repo brought here. We can delete that repo after this is done.

This is superior since as the API changes it will inform us that the client needs to be updated too.